disco/sources/base.py

```diff
@@ -488,8 +497,26 @@ def replace_func(match):
                 print(line, end="")
     
     @staticmethod
-    def _exclude_load_profile(workspace):
-        """Exclude load profile from loads model"""
+    def _strip_shape_files_from_master_file(workspace):
+        """Exclude load and pv shapes from master DSS model file"""
+        new_lines = []
+        with open(workspace.master_file, "r") as f:
+            for line in f.readlines():
+                lowered = line.strip().lower()
+                if lowered.startswith("!"):
+                    new_lines.append(line)
+                    continue
+                elif "loadshapes.dss" in lowered or "pvshapes" in lowered:
```

Comment:
Would you mind adding "redirect" here to be more explicit?

Reply:
Updated.